test/clj/milia/api/charts_test.clj

```diff
@@ -22,4 +25,37 @@
          (chart dataset-id :field-name) => :some-chart
          (provided
           (make-url "charts" suffix-with-field) => url
-          (parse-http :get url) => :some-chart)))
+          (parse-http :get url) => :some-chart))
+
+  (facts "About get chart data"
```

Comment:
The get function was not tested. Added tests for all the three possible scenarios

Reply:
small change that might be needed though: the assertions should be inside the `fact` statements

Reply:
For example [(from the documentation here)](https://cljdoc.org/d/midje/midje/1.9.3-alpha1/api/midje.sweet#fact)

`(fact "one plus one is two" (+ 1 1) => 2)`